packages/protocol/test/team/TimelockTokenPool.t.sol

```diff
@@ -9,32 +9,44 @@ contract MyERC20 is ERC20 {
     }
 }
 
+contract USDC is ERC20 {
+    constructor(address recipient) ERC20("USDC", "USDC") {
+        _mint(recipient, 1_000_000_000e6);
+    }
+
+    function decimals() public view virtual override returns (uint8) {
```

Comment:
this is not used at all.

Reply:
It is function overloading, so that USDC (with 6 decimals) works as expected. If you change this part, test will fail.